tb_plugin/examples/resnet50_autograd_api.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
Why was this added back in?

Reply:
@ilia-cher are we OK with this being here?

Reply:
@gdankel i assume this is just an example for the tb plugin, was there some other concern?

Reply:
since new profiler api will still be prototype or beta in PT 1.8, so I assume many users may still use the old profiler api. Not adding the autograd profiler example may make user feel that we don't support it. So I feel it is better to add an example.

Reply:
we will remove this example after autograd profiler is officially deprecated in future release.